### ISSUE TITLE
Getting strange output

### ISSUE THREAD

**Issue by WXuanyang** [kwea123/nerf_pl#113]
I'm getting those strange output with my own data:
![image](https://user-images.githubusercontent.com/69446297/144478002-81688b5d-0b2d-4740-825e-6f7a6d5fd405.png)
![image](https://user-images.githubusercontent.com/69446297/144478120-14c6f135-eaa1-4edc-a221-26a286413301.png)

Here's my setting:
Branch: dev
python train.py \
   --dataset_name llff \
   --root_dir data/$SCENE \
   --N_importance 64 --img_wh 440 360 \
   --num_epochs 30 --batch_size 1024 \
   --optimizer adam --lr 5e-4 \
   --lr_scheduler steplr --decay_step 10 20 --decay_gamma 0.5 \
   --exp_name $SCENE \
   --spheric

And my inputs in colmap:
![colmap](https://user-images.githubusercontent.com/69446297/144478347-a9dca3bd-3a77-4339-a9db-df506a2259ea.png)

The psnr also looks strange, the train psnr grows rapidly in first 2 epochs and stablilzed to around 30, but val psnr got only 12.5-13.3

Wish anyone can give some suggestions

**Comment by kwea123** [comment]
Can you show the tensorboard training log (the image tab)?
Did you add `--spheric_poses` in evaluation?

**Comment by WXuanyang** [comment]
> Can you show the tensorboard training log (the image tab)? Did you add `--spheric_poses` in evaluation?

Many thanks for your quick reply!

I just noticed that I forgot the spheric-poses in evaluation, but I'm afraid that's not the casue of my problem.
The training log shows that something is clearly far wrong, as the eval psnr isn't increasing at all
![log](https://user-images.githubusercontent.com/69446297/144535252-090045c5-9caa-4f35-8b3c-f0019a2590aa.png)
![individualImage](https://user-images.githubusercontent.com/69446297/144535554-e781bdd9-d8db-46f8-b612-d423482ccbdc.png)

And here's an example of the result with the spheric-poses flag
![image](https://user-images.githubusercontent.com/69446297/144535463-e8598d52-75b2-43ff-a424-05ae4f718ae1.png)

This problem is happening with or without the spheric flag. Also I've also tried some of the llff examples with the same setting but the training goes pretty well, so I'm just thinking if there's something wrong with my data or colmap reconstruction
